pkg/webhook/validate_test.go

```diff
@@ -159,7 +159,7 @@ func TestValidator_Handle(t *testing.T) {
 				core.Annotation(csmetadata.DeclaredFieldsKey, `{"f:metadata":{"f:labels":{"f:app.kubernetes.io/managed-by":{}},"f:annotations":{"f:configmanagement.gke.io/managed":{},"f:configsync.gke.io/manager":{}}},"f:rules":{}}`),
 			),
 			user: configSyncNamespaceReconciler("bookstore", repoSyncName),
-			deny: metav1.StatusReasonUnauthorized,
+			deny: metav1.StatusReasonForbidden,
```

Comment:
Do you know why the deny code is changed? Forbidden is quite different from unauthorized.

Reply:
It looks like the logic is changed in controller runtime. To be fair `forbidden` is more accurate than `unauthorized` in this scenario:

- Forbidden means that the user has valid credentials but is not allowed to perform the action
- Unauthorized means that the user does not have valid credentials

Reply:
make sense